tests/rust/test_secret_key.py

```diff
@@ -0,0 +1,39 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+
+import pytest
+
+
+@pytest.mark.rust
+def test_secret_key():
+    from parsec.crypto import _PySecretKey, _RsSecretKey, SecretKey
+
+    assert SecretKey is _RsSecretKey
+
+    KEY = b"a" * 32
+    MESSAGE = b"test secret key"
+
+    rst = SecretKey(KEY)
+    py = _PySecretKey(KEY)
+
+    assert isinstance(rst, SecretKey)
+    assert isinstance(py, _PySecretKey)
+
+    assert rst.secret == py.secret
+    assert repr(rst) == repr(py)
+
+    ciphered_rst = rst.encrypt(MESSAGE)
+    ciphered_py = py.encrypt(MESSAGE)
+
+    # We cannot assert "ciphered_rst == ciphered_py" because the encryption is not exactly the same
+
+    # Verify with both decryption
+    assert rst.decrypt(ciphered_rst) == py.decrypt(ciphered_py)
+    assert rst.decrypt(ciphered_py) == py.decrypt(ciphered_rst)
+
+    # TO-DO: test for hmac
+    # first we need to implement hmac in rust
+    # assert rst.hmac(data, digest_size) == py.hmac(data)
```

Comment:
Seems like `hmac` is implemented now :)

Reply:
`    // TODO
    pub fn hmac(&self, _data: &[u8], _digest_size: usize) -> Vec<u8> {
        // // blake2b(data, digest_size=digest_size, key=self, encoder=RawEncoder)
        // let key = blake2b::Key::from_slice(&self.0);
        // blake2b::derive_from_key(
        //     subkey: &mut [u8],
        //     subkey_id: u64,
        //     ctx: [u8; 8],
        //     key,
        // );
        vec![]
    }`
Not yet unfortunatly

Reply:
done